Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM continuumio/miniconda3
+
+COPY . /descartes-rpa/
```

Comment:
change  naming convention from descartes-rpa to descartes_rpa (in all code)

Reply:
Do we change the repo name as well? I've already changed the rest

Reply:
@SolomonShorser-OICR already changed it : )

Reply:
Thanks @teslajoy and @SolomonShorser-OICR :D